packages/generator/src/generator-options.ts

```diff
@@ -285,11 +292,17 @@ export const generatorOptionsCli = {
     deprecated:
       "Since v2.12.0. Use the 'include' option to add your own package.json file instead."
   },
-  generateJs: {
+  transpile: {
     describe:
-      'By default, the generator will also generate transpiled .js, .js.map, .d.ts and .d.ts.map files. When set to false, the generator will only generate .ts files.',
+      'Transpile the generated TypeScript code. When enabled a default `tsconfig.json` will be generated and used. It emits `.js`, `.js.map`, `.d.ts` and `.d.ts.map` files. To configure transpilation set `--tsconfig`.',
```

Comment:
[req] => as before: `.d.ts.map`

Reply:
[q] Here you wrote `--tsconfig` although currently it is called `tsConfig`. Does that work?